### DISCUSSION TITLE
Feedback from a packager

### DISCUSSION THREAD

**Discussion by Mic92** [rizinorg/rizin#937] [Archive]
I just upgraded cutter to 2.0 and also included rizin in the process into [nixpkgs](https://github.com/NixOS/nixpkgs/)
and it is a lot easier to package than radare2 before where we needed 
a python script to get all the versions/git refs right.
I also appreciate that it allows to use system libraries for most things including capstone.
PR is here https://github.com/NixOS/nixpkgs/pull/118041

**Comment by wargio** [comment]
Thank you for this feedback. it's extremely useful for us since we can understand if the direction we have taken is good or not.
Thank you.

Deroad.

**Comment by ITAYC0HEN** [comment]
Thank you very much @Mic92! We are trying hard to make the packaging process of Cutter and Rizin as easy as possible. Your feedback is fantastic and encourages us to keep on going 💜

Please don't hesitate to share more feedback and suggest improvements.

Once again, thank you very very much!
